src/modules/registrypreview/RegistryPreviewUI/RegistryPreviewXAML/MainWindow.xaml.cs

```diff
@@ -49,7 +49,7 @@ internal MainWindow()
             IntPtr windowHandle = WinRT.Interop.WindowNative.GetWindowHandle(this);
             WindowId windowId = Win32Interop.GetWindowIdFromWindow(windowHandle);
             appWindow = Microsoft.UI.Windowing.AppWindow.GetFromWindowId(windowId);
-            appWindow.SetIcon("app.ico");
+            appWindow.SetIcon("Assets\\RegistryPreview\\app.ico");
```

Comment:
Should this have ../ at the beginning?

Reply:
XAML have their own Assets folder as well. Why are you thinking about "../" here?